frontend/tailwind.config.js

```diff
@@ -86,10 +86,12 @@ module.exports = {
         600: "#319795",
         700: "#2c7a7b",
         800: "#285e61",
-        900: "#234e52"
+        900: "#234e52",
+        150: "#144656"
```

Comment:
I don't think you are using these shades at the moment, so I would remove them until they are needed again